app/src/components/market/market_wizard_creator.tsx

```diff
@@ -70,6 +56,7 @@ export const MarketWizardCreator = (props: Props) => {
         probability: 50,
       },
     ],
+    questionIsFromRealitio: false,
```

Comment:
Maybe `existingQuestion` or `loadedQuestion` to make it less coupled to Realitio.

Reply:
Done!